components/Layout/Navbar.tsx

```
@@ -26,63 +27,121 @@ import {
-        
+    <>
+      {/* Skip Navigation Link for Screen Readers */}
+      <a
+        href="#main-content"
+        className="sr-only focus:not-sr-only focus:absolute focus:top-4 focus:left-4 bg-primary text-primary-foreground px-4 py-2 rounded z-50"
+      >
+        Skip to main content
+      </a>
+
+      <header className={cn("w-full border-b bg-secondary/15 backdrop-blur supports-[backdrop-filter]:bg-secondary/15 sticky z-40", className)}>
+        <nav className="w-full flex h-16 items-center" role="navigation" aria-label="Main navigation">
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<nav>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)